content/en/account_management/rbac/permissions.md

```diff
@@ -0,0 +1,372 @@
+| monitors_read     | Ability to view monitors                     | false    |
+| monitors_write    | Ability to change, mute, and delete monitors | false    |
+| monitors_downtime | Ability to set downtimes for your monitors   | false    |
+
+### Log Management
+
+| Name                         | Description                                | Scopable |
+| ---------------------------- | ------------------------------------------ | -------- |
+| logs_read_index_data         | Read a subset of all log indexes           | true     |
+| logs_modify_indexes          | Update the definition of log indexes       | false    |
+| logs_live_tail               | Access the live tail feature               | false    |
+| logs_write_exclusion_filters | Update a subset of the exclusion filters   | true     |
+| logs_write_pipelines         | Update a subset of the log pipelines       | true     |
+| logs_write_processors        | Update the log processors in an index      | true     |
+| logs_write_archives          | Update the external archives configuration | false    |
+| logs_public_config_api       | Access the Logs Public Config API (r/w)    | false    |
+| logs_generate_metrics        | Access the Generate Metrics feature        | false    |
+
+More details about these permissions below.
+
+#### Configuration
+
+##### logs_live_tail
+
+Grants a role the ability to use the live tail feature.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `read` as below for the wanted role:
+{{< img src="account_management/rbac/logs_livetail_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_generate_metrics
+
+Grants a role the ability to use the Generate Metrics feature. This permission is global and applies to the configuration of all the metrics generated from logs.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_generate_metrics_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_modify_indexes
+
+Grants a role the ability to create and modify log indexes. This includes:
+
+- Setting inclusion queries for which logs should be routed into an index.
+- Setting log retention for an index.
+- Limiting which roles have read access on an index (`logs_read_index_data`).
+- Which roles can modify exclusion filters for an index (`logs_write_exclusion_filters`).
+
```

Comment:
This does not list all that the `logs_modify_indexes` permission does -- it also implicitly grants the same read- and write-access as some other permissions. 

In the old docs, we included this note:

> **Note:** This permission also grants read access on all log indexes and write permissions on all index exclusion filters, since any role that can modify indexes also can grant itself these additional permissions.

And in fact it also grants live-tail read access to.

I think this is important information to document, or else users will get very confused (at best) or grant too much read access without realizing it (at worst).